index.js

```diff
@@ -4,7 +4,7 @@ var os = require('os');
 var execSync = require('child_process').execSync;
 
 module.exports = function() {
-  return module.exports.os() || module.exports.env() || module.exports.execSync();
+  return module.exports.env() || module.exports.os() || module.exports.execSync();
```

Comment:
Given that this isn't 100% obvious, mind adding a short comment describing why we prefer one over the other?